### DISCUSSION TITLE
Could we specify some message IDs for downloading?

### DISCUSSION THREAD

**Discussion by poppinlp** [Dineshkarthik/telegram_media_downloader#338] [Q&A]
Right now, we have `last_read_message_id` to specify the start point, but it will automatically download all the following stuff.
Could we just specify some message IDs for downloading? I reckon that would be much useful.

**Comment by tangyoha** [comment]
@poppinlp hi,
you can create a map  which you want to download message id.
Skip which you want not to download message.

**Reply by poppinlp** [reply]
haha, thank you.
I've modified the code to add a "limit" config, so I can easily download from a message ID with a limitation count.

**Reply by amitsha1** [reply]
can u please suggest me how to do this download videos by massage id

**Reply by tangyoha** [reply]
see https://github.com/tangyoha/telegram_media_downloader
我们添加了下载过滤器可以指定下载 https://github.com/tangyoha/telegram_media_downloader/wiki/%E5%A6%82%E4%BD%95%E4%BD%BF%E7%94%A8%E8%BF%87%E6%BB%A4%E5%99%A8
